src/components/account/details/SideMenu.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react'
+
+import '../../../styles/account/details/side_menu.css'
+
+export default class SideMenu extends Component {
+  render() {
+    return (
+      <div className="side-menu">
+        <div className="side-menu-list">
+          <div className="side-menu-list-item active">Account</div>
+          <div className="side-menu-list-item"><a href="#">Dashboard</a></div>
```

Comment:
There's some inconsistent formatting among your files in this PR - embedded elements on a single line (which I'm fine with if its < 80 chars), as well as embedded elements on their own line, indented.  Please select one approach and apply it consistently (the new line thing is probably a little more readable, but I'm generally neutral on the question if we satisfy the character limit constraint).